### PR TITLE
Michael/#ORC-406/helm-update-versions

### PR DESCRIPTION
**Pull request by mclacore**

### CODE REVIEW

File: src/main.tf [hunk 1]
```diff
@@ -55,11 +55,10 @@ resource "azurerm_kubernetes_cluster" "main" {
   the customer might set as their VNet CIDR. These are also the defaults for
   these parameters when deploying AKS in the Azure Portal. */
   network_profile {
-    network_plugin     = "azure"
-    network_policy     = "azure"
-    dns_service_ip     = "172.20.0.10"
-    docker_bridge_cidr = "172.17.0.1/16"
```
Comment (chrisghill) [inline src/main.tf:61]: Why did you remove the `docker_bridge_cidr`?  Is this a destructive change?

Comment (mclacore) [reply src/main.tf:61]: Good question! Not a destructive change. Redeployed AKS in personal org after republishing. 

The reason why it's being removed is because the azure terraform provider were on now has marked it as deprecated and no longer needed.

Comment (mclacore) [reply src/main.tf:61]: > Make sure you double check that this change isn't destructive to existing clusters.

`terraform plan` for this confirms no changes, and also didn't have any changes when deploying in the UI.

**Review by chrisghill — APPROVED** [review]
